apollo-adapters-core/src/commonMain/kotlin/com/apollographql/adapter/core/UnitAdapter.kt

```diff
@@ -0,0 +1,16 @@
+package com.apollographql.adapter.core
+
+import com.apollographql.apollo.api.Adapter
+import com.apollographql.apollo.api.CustomScalarAdapters
+import com.apollographql.apollo.api.json.JsonReader
+import com.apollographql.apollo.api.json.JsonWriter
+
+internal object UnitAdapter : Adapter<Unit> {
```

Comment:
Sorry I missed it before, this shouldn't be `internal`. That also explains why `apiDump` doesn't include it.

Reply:
Oh yeah that explains it! Sorry, bad copy/paste :see_no_evil: 

It should be fixed now :+1: